src/gcp/firestore.ts

```diff
@@ -1,18 +1,16 @@
-"use strict";
+import api = require("../api");
```

Comment:
I never know when to do this or do `import * as api from "../api"` ...

Reply:
the latter is definitely preferred, but when I migrate js -> ts I prefer to keep the changes minimally invasive